code/game/objects/structures/grille.dm

```diff
@@ -70,13 +70,15 @@
 		var/mob/M = AM
 		health -= 10
```

Comment:
They still cause damage?

Reply:
that's for mobs, not clouds

Reply:
I just made the visible message not display if the mob was invisible. Maybe I should add a different message.